src/backend/distributed/transaction/transaction_management.c

```diff
@@ -328,12 +328,15 @@ CoordinatedTransactionCallback(XactEvent event, void *arg)
 				CoordinatedRemoteTransactionsAbort();
 			}
```

Comment:
I'm wondering if we should do something similar for case XACT_EVENT_PARALLEL_ABORT:. What do you think @thanodnl ? How can we trigger that code-path ?

Reply:
Currently there is no body in `XACT_EVENT_PARALLEL_ABORT`.
My feeling is that the block is unreachable due to how the citus planner works.

I am inclined to leave that for now and not add anything there without more signals it causes issues.
If the block is reachable I think we should perform more work than simply the change we are making here.